index.html

```diff
@@ -278,6 +278,71 @@ <h3>
           reset because its [=permission/lifetime=] has expired.
         </p>
       </section>
+      <section>
+        <h3>
+          Permission Store
+        </h3>
+        <p>
+          The user agent maintains a single <dfn class="export">permission store</dfn> which is a [=/list=] of [=permission store entries=].
```

Comment:
Probably :)

Reply:
Hmm having tried this now I realize that using a map is a major change to the concepts we've been proposing in this PR as of now. It's a departure from saying e.g. "get a permission store entry" to something like "get the permission state for a permission store key", where the key would now be (description, key) and we might have to rename the latter key to something like "scope".

Overall I'm not a fan of that new map approach personally, and wanted to get thoughts on this change before writing it all up :)

Would you be comfortable with just using a set as an alternative to ensure this invariant?

(cc @annevk)

Reply:
I'm still digesting the PR, but I think a set would work?

Reply:
Ah, yeah, that makes sense, thanks @jyasskin. I think since this is largely about spec correctness and shouldn't affect implementations too much, we can also revisit this later.